packages/backend/src/accounting/service.ts

```diff
@@ -52,7 +54,7 @@ export interface Deposit {
 }
 
 export interface Withdrawal extends Deposit {
-  timeout: bigint
+  timeout?: bigint
```

Comment:
This (and other) `timeout` parameters should be named with the unit; e.g. `timeoutNano`, `timeoutMilli`.
JS timeouts are typically milliseconds (e.g. `setTimeout`) but TigerBeetle uses nanoseconds. Since we need to use both (in different contexts), I think its worth disambiguating them in the property name so that there's no confusion.

Reply:
What if we kept the name `timeout` but keep it consistent as milliseconds and convert to nano-seconds where passing to tigerbeetle node?

Reply:
:+1: That works.